packit/cli/update_dist_git.py

```diff
@@ -131,4 +133,5 @@ def update_dist_git(
         commit_msg=message,
         sync_default_files=False,
         pkg_tool=pkg_tool,
+        mark_commit_origin=True,
```

Comment:
I'm wondering if we want to make this configurable for users - I guess it would complicate things on our side and in the end, people can still commit whatever they want. So forcing the trailers is showing a good practice which has only upsides.

Reply:
Right now, I don't think we want this. The only reason this flag was introduced, is b/c `update_dist_git()` is used as part of `sync_release()`, where in a Packit context we don't want these trailers to be included.

Reply:
right, understood